File: src/main/java/io/jsql/hazelcast/test/Mhazelcasttest.java
package io.jsql.hazelcast.test;

import io.jsql.hazelcast.Mhazelcast;
import io.jsql.hazelcast.SqlUpdateLog;

/**
 * Created by 长宏 on 2017/5/5 0005.
 */
public class Mhazelcasttest {
    public static void main(String[] args) {
        Mhazelcast mhazelcast = new Mhazelcast();
        mhazelcast.sqlUpdateLogs.add(new SqlUpdateLog(1, "sql"));

        System.out.println(mhazelcast.sqlUpdateLogs.size());
    }
}

File: src/main/java/io/jsql/hazelcast/test/Mhazelcasttest.kt
/*
 * Java-based distributed database like Mysql
 */

package io.jsql.hazelcast.test

import io.jsql.hazelcast.MyHazelcast

/**
 * Created by 长宏 on 2017/5/5 0005.
 */
object Mhazelcasttest {
    @JvmStatic
    fun main(args: Array<String>) {
        val mhazelcast = MyHazelcast()
        mhazelcast.exeSql("create database db3", "d")
    }
}
